### PR TITLE
Feature/extensions

### PR DESCRIPTION
**Pull request by hvohr**
What I did: Added a function that shuffles the front page recipes. Clicking the title also brings the user back to home.  I also added shuffle function testing. 

Did this break anything?
- [x] No
- [ ]  Yes

Type of change
- [x] New feature (non-breaking change which adds functionality)
- [x]  Bug fix (non-breaking change which fixes an issue)

Checklist:
- 3 tests are still failing (Issue will be fixed in the future)
- [x]  The code runs locally
- [x]  The code is DRY. If it's not, I tried my best
- [x]  I reviewed my code before pushing

What's next? 
Adding the viewing recipe functionality on the front page pictures and removing the unnecessary tests from the file. Implementing Iteration 2 as well.

### CODE REVIEW

File: src/domUpdates.js [hunk 1]
```diff
@@ -32,6 +33,13 @@ allRecipeDisplay.addEventListener('click', function (event) {
   viewSelectedRecipe(event);
 });
 
+homeButton.addEventListener('click', function () {
+  showHomePage()
+  randomizeHomePage()
+})
```
Comment (andreasorensen) [inline src/domUpdates.js:39]: Love that this works to randomize the front pics too!

File: src/domUpdates.js [hunk 2]
```diff
@@ -46,13 +54,45 @@ function showSearchResults() {
...
+}
+
 function renderFilteredRecipes() {
   const tags = Array.from(checkCategories).filter((category) => category.checked).map(c => c.id)
   if (tags.length === 0) {
     showRecipes()
     return
   }
     let filtered = recipesFromTag(recipeData, tags);
+    console.log(recipeData)
```
Comment (andreasorensen) [inline src/domUpdates.js:95]: We should delete this console.log

File: src/recipeUtils.js [hunk 3]
```diff
@@ -49,13 +49,17 @@ const calculateRecipeCost = (recipe, ingredients) => {
    return instructions.join(' ')
  };
 
-
+function shuffleData(recipes) {
+  recipes.sort(() => Math.random() - 0.5)
```
Comment (andreasorensen) [inline src/recipeUtils.js:53]: LOVE THIS FUNCTION!

File: src/recipeUtils.js [hunk 4]
```diff
@@ -49,13 +49,17 @@ const calculateRecipeCost = (recipe, ingredients) => {
...
+  return
+}
```
Comment (Denzel-Turambi) [inline src/recipeUtils.js:55]: Nice function! short and sweet!

**Review by Denzel-Turambi — APPROVED** [review]

**Review by andreasorensen — APPROVED** [review]
Thanks for taking care of the front page functionality. 
I'll be sure to add the viewSelectedRecipe() to the randomizeHomePage function once I fix the bug.

File: src/domUpdates.js [hunk 5]
```diff
@@ -46,13 +54,45 @@ function showSearchResults() {
     </div>`)
 };
 
+
+function showHomePage() {
+  addHiddenClass([allRecipeDisplay], [allFilterDisplay]);
+  removeHiddenClass([frontRecipeDisplay]);
+}
+
+function randomizeHomePage() {
```
Comment (fernandorobles97) [inline src/domUpdates.js:63]: Solid function!

**Review by fernandorobles97 — APPROVED** [review]

**Pull request merged**